catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/sharing/ShareableMetacardAttributes.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard.sharing;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.types.Core;
+import java.util.Map;
+import java.util.Set;
+
+public class ShareableMetacardAttributes implements MetacardType {
+
+  public static final String NAME = "shareable";
+
+  public static final String FORM_SHARING = "metacard.sharing";
+
+  private static final Map<String, AttributeDescriptor> DESCRIPTORS =
+      ImmutableMap.of(
+          Core.METACARD_OWNER,
+          new AttributeDescriptorImpl(
+              Core.METACARD_OWNER,
+              true /* indexed */,
+              true /* stored */,
+              true /* tokenized */,
+              false /* multivalued */,
+              BasicTypes.STRING_TYPE),
+          FORM_SHARING,
```

Comment:
❓ How will this XML attribute be used? I don't think the workspace sharing XML attribute is used by its policy.